src/ProfileView.jl

```diff
@@ -233,67 +234,64 @@ function viewgui(fcolor, gdict::NestedGraphDict; data=nothing, lidict=nothing, w
+        nb_tasks = GtkNotebook() # for holding the per-task pages
+        Gtk4.scrollable(nb_tasks, true)
+        Gtk4.show_tabs(nb_tasks, length(task_tabs) > 1)
         task_tab_num = 1
         for task_tab in task_tabs
             g = gdict_thread[task_tab]
             gsig = Observable(g)  # allow substitution by the open dialog
             c = canvas(UserUnit)
-            set_gtk_property!(widget(c), :expand, true)
-
-            f = Frame(c)
-            tb = Toolbar()
-            tb_open = ToolButton("gtk-open")
-            Gtk.GAccessor.tooltip_text(tb_open, "open")
-            tb_save_as = ToolButton("gtk-save-as")
-            Gtk.GAccessor.tooltip_text(tb_save_as, "save")
-            tb_zoom_fit = ToolButton("gtk-zoom-fit")
-            Gtk.GAccessor.tooltip_text(tb_zoom_fit, "zoom to fit")
-            tb_zoom_in = ToolButton("gtk-zoom-in")
-            Gtk.GAccessor.tooltip_text(tb_zoom_in, "zoom in")
-            tb_zoom_out = ToolButton("gtk-zoom-out")
-            Gtk.GAccessor.tooltip_text(tb_zoom_out, "zoom out")
-            tb_info = ToolButton("gtk-info")
-            Gtk.GAccessor.tooltip_text(tb_info, "ProfileView tips")
-            tb_text_item = ToolItem()
-            Gtk.GAccessor.expand(tb_text_item, true)
-            tb_text = Entry()
-            Gtk.GAccessor.has_frame(tb_text, false)
-            Gtk.GAccessor.sensitive(tb_text, false)
-            push!(tb_text_item, tb_text)
+            set_gtk_property!(widget(c), :vexpand, true)
+
+            f = GtkFrame(c)
+            tb = GtkBox(:h)
+            tb_open = GtkButton(:icon_name,"document-open-symbolic")
+            Gtk4.tooltip_text(tb_open, "open")
+            tb_save_as = GtkButton(:icon_name,"document-save-as-symbolic")
+            Gtk4.tooltip_text(tb_save_as, "save")
+            tb_zoom_fit = GtkButton(:icon_name,"zoom-fit-best-symbolic")
+            Gtk4.tooltip_text(tb_zoom_fit, "zoom to fit")
+            tb_zoom_in = GtkButton(:icon_name,"zoom-in-symbolic")
+            Gtk4.tooltip_text(tb_zoom_in, "zoom in")
+            tb_zoom_out = GtkButton(:icon_name, "zoom-out-symbolic")
+            Gtk4.tooltip_text(tb_zoom_out, "zoom out")
+            tb_info = GtkButton(:icon_name, "dialog-information-symbolic")
+            Gtk4.tooltip_text(tb_info, "ProfileView tips")
+            tb_text = GtkEntry()
+            Gtk4.has_frame(tb_text, false)
+            Gtk4.sensitive(tb_text, false)
+            push!(tb, tb_text)
 
             push!(tb, tb_open)
             push!(tb, tb_save_as)
-            push!(tb, SeparatorToolItem())
+            push!(tb, GtkSeparator(:h))
             push!(tb, tb_zoom_fit)
             push!(tb, tb_zoom_out)
             push!(tb, tb_zoom_in)
-            push!(tb, SeparatorToolItem())
+            push!(tb, GtkSeparator(:h))
             push!(tb, tb_info)
-            push!(tb, SeparatorToolItem())
-            push!(tb, tb_text_item)
+            push!(tb, GtkSeparator(:h))
```

Comment:
```suggestion
            push!(tb, GtkSeparator(:h))
            push!(tb, tb_text)
```
Moving `push!(tb, tb_text)` preserved the order of the toolbar.